### ISSUE TITLE
Feature Request: Mouse-over tooltip for more `Parameters/Properties` and potential bug fix

### ISSUE THREAD

**Issue by r-heimann** [aws-cloudformation/cfn-lint#4481]
### Is this feature request related to a new rule or cfn-lint capabilities?

New capability

### Describe the feature you'd like to request

I noticed that cfn-lint is unable to display information / display incorrect information for the Parameter configurations.

For example, the config for `Type` displays a tooltip:

```yaml
AWSTemplateFormatVersion: 2010-09-09
Description: Test


Parameters:
  InstanceTypeParameter:
    Description: Enter t2.micro, m1.small, or m1.large. Default is t2.micro.
    Type: String
    Default: t2.micro
    AllowedValues:
      - t2.micro
      - m1.small
      - m1.large


Resources:
  S3Bucket:
    Type: AWS::S3::Bucket
    Properties:
      BucketName: !Ref InstanceTypeParameter
```

<img width="493" height="203" alt="Image" src="https://github.com/user-attachments/assets/b0717c49-ce60-491f-bf80-9a6a4f888270" />

But according to the documentation more values are allowed:

https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/parameters-section-structure.html

After changing it to a false value it displays the correct tooltip:

<img width="752" height="323" alt="Image" src="https://github.com/user-attachments/assets/f4f2ce0b-53d6-4b1e-8201-61afb3e2a620" />

The other values do not seem to display any kind of tooltip.

### Describe the solution you'd like

cfn-lint is able to display up-to-date information on all of the currently supported CloudFormation `Parameters` configurations.

### Is this something that you'd be interested in working on?

- [ ] 👋 I may be able to implement this feature request

### Would this feature include a breaking change?

- [ ] ⚠️ This feature might incur a breaking change
